en/docs/administer/managing-users-and-roles/managing-permissions.md

```diff
@@ -157,7 +157,13 @@ Additionally, all permissions listed under <strong>Admin/Monitor</strong> in the
 </tbody>
 </table>
 
-## Adding role-based permissions
+## Adding scope-based permissions
+
+WSO2 API-Manager (APIM) publisher, dev portal, and admin portal UIs work on top of APIM-specific Rest APIs. Access to these APIs is given based on OAuth token scopes. When a user logs in to the publisher or dev portal, the access token is generated with a specific set of scopes based on the user's roles.
```

Comment:
WSO2 API Manager (WSO2 API-M) API Publisher, Developer Portal, and Admin Portal UIs work on top of API-M specific REST APIs. OAuth token scopes provide a way to access these APIs. When a user logs in to any  Portal, it generates an access token with a specific set of scopes based on the user's roles.